projects/cds/src/merchandising/model/merchandising-products.model.ts

```diff
@@ -1,6 +1,10 @@
 import { Product } from '@spartacus/core';
 
+export interface MerchandisingProduct extends Product {
+  metadata?: Map<string, string>;
+}
+
 export interface MerchandisingProducts {
   metadata?: Map<string, string>;
```

Comment:
just for my understanding, 

there's a global metadata for 'products'
and a lower level metadata for a single 'product'

Reply:
That's right